CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Coding style
+
+* like HOL source code
+  - Spaces, no tabs
+  - No unicode
+  - `snake_case` (e.g. `bir_number_of_mem_splits_def`)
+
+
+
+# Branch policy
+
+## `master` branch
+
+`master` is the branch where every feature is available, but not necessarily finalized:
+ - Can cheat, but has to be avoided (allowed only in a small number of modules, which are explicitly marked unstable, no-cheat CI test mentioned below would need a corresponding exception list)
+ - Code should not be be commented out
+ - Our Travis CI must pass
+   - **Holmake must work** (i.e. must correctly compile)
+   - All tests must succeed
+ - Bug-fixing commits are ok
+ - At least 1 accepting review is needed in order to merge into `master`
+
+Notice:
+ - **No development happens on the `master` branch**, but rather on separate feature branches
+ - **In order to prevent mayhem**, define good interfaces for your code (so that development won't break existing code)
+
+Follow these instructions whenever you merge to master:
+  - `grep` for "cheat" (TODO: add this as a CI test)
```

Comment:
Prefer a new issue instead of this TODO that we will forget.

Reply:
done, and kept as TODOs. pushing now